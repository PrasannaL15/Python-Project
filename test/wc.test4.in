This    text    has    multiple    spaces    and    tabs    between    words, testing how well the word count utility handles different whitespace characters.

