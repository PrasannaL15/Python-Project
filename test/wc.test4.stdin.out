       2      20     160